### PR TITLE
Shift4: Capture field changes and hardcode header values

### PR DESCRIPTION
**Pull request by ajawadmirza**
Capture changes to add card present field, use same invoice number as authorization, and hardcode header values.

SER-223
SER-226
SER-222

Remote:
16 tests, 37 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

Unit:
5273 tests, 76175 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

Rubocop:
746 files inspected, no offenses detected

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -85,6 +85,7 @@
 * Worldpay: Add machine cookie to subsequent calls during 3DS challenge [mbreenlyles] #4513*
 * Shift4: Scrub `securityCode` fix [naashton] #4524
 * Credorax: Update `OpCode` for credit transactions [dsmcclain] #4279
+* Shift4: add card `present` field, use previous transaction authorization for capture, and hardcode header values [ajawadmirza] #4528
```
Comment (drkjc) [inline CHANGELOG:88]: Capitalize the `a` in `add`

**Review by drkjc — CHANGES_REQUESTED** [review]
@ajawadmirza Sorry to make you do this, but there's a lowercase word in the CHANGELOG that needs to be capitalized. Otherwise, everything looks great!

File: CHANGELOG (hunk 1 above)
Comment (ajawadmirza) [reply CHANGELOG]: Thanks, change is updated 👍

**Comment by ajawadmirza** [comment]
I have also added card present field for refund as this ticket (SER-222) is also related to this PR. 

https://github.com/activemerchant/active_merchant/pull/4528/files#diff-fc67ccfe1e4c5ec05676588bd97cee17d72d6ea9a1f84ede7c2f98e0f947e8c9R119

**Review by drkjc — APPROVED** [review]
Nice work, Jawad. Looks great.

**Pull request closed without merging**